### PR TITLE
Add Population Bomb animation

### PR DESCRIPTION
**Pull request by kittenchilly**
## Description
Adds an animation for Population Bomb, a cut that randomly starts from one side or the other.

## Images
popbomb
Ignore the garbage text, that's a separate bug that I'll fix in another PR.

## **Discord contact info**
kittenchilly

### CODE REVIEW

File: src/battle_anim_new.c [hunk 1]
```diff
@@ -9285,3 +9285,13 @@ static void AnimMakingItRain(struct Sprite *sprite)
     sprite->callback = TranslateSpriteInEllipse;
     sprite->callback(sprite);
 }
+
+void AnimTask_RandomBool(u8 taskId)
+{
+    if (Random() % 2 == 1)
```
Comment (pkLucid) [inline src/battle_anim_new.c:9291]: use `RandomPercentage`

Comment (AlexOn1ine) [inline src/battle_anim_new.c:9291]: Use RandomPercentage

Comment (AlexOn1ine) [inline src/battle_anim_new.c:9291]:
```suggestion
    if (RandomPercentage(RNG_NONE, 50))
```

**Comment by kittenchilly** [comment]
Reviews addressed

**Review by AlexOn1ine — APPROVED** [review]

**Pull request merged**